### ISSUE TITLE
Showing empty space warning when there is no empty space in the code

### ISSUE THREAD

**Issue by ghost** [dotnet/roslyn#27900]
(bool) /\ 0

This is the invalid c# code, first it's have invalid expression '/' and second is '\'
when I am executing the code it's showing 4 issues.

I know this code is terribly incorrect but it should only show 3.  I am thinking that invalid expression term
'' should not be shown in the errors because there is no empty space there at the 1,9.

![image](https://user-images.githubusercontent.com/2105258/41454322-92e708d2-7096-11e8-89ea-3e8c9fa052c3.png)

**Comment by CyrusNajmabadi** [comment]
@anirugu Can you please close dotnet/csharplang#1637 and just copy your full repo here?  This is a compiler issue and belongs in dotnet/roslyn not dotnet/csharplang.  Thanks!

**Comment by anirugu** [comment]
yes, I close the issue, I am not sure how to copy so I edit that and copy the same content here.

**Comment by CyrusNajmabadi** [comment]
Closing out as too minor.
